README.md

```diff
@@ -175,6 +175,109 @@ include::alice-bob.puml[]
 
 ![PlantUML diagram](https://kroki.io/plantuml/png/eNpzKC5JLCopzc3hSszJTE5V0LVTSMpP4nJIzUsBCgIApPUKcg==)
 
+### Syntax
+
+You can declare positional and named attributes when using the block or the macro form.
+
+**Positional attributes**
+
+When using the block form:
+
+1. The first positional attribute specifies the diagram type (see below for a complete list).
+2. The second optional positional attribute assigns a file name (i.e. target) to the generated diagram. *Currently, the value of this attribute is ignored, and an auto-generated hash will be used as file name for caching purposes (see #48)*.
+3. The third optional positional attribute specifies the image format.
+
+Example:
+
+```
+[mermaid,abcd-flowchart,svg]
+....
+graph TD;
+    A-->B;
+    A-->C;
+    B-->D;
+    C-->D;
+....
+```
+
+In the above example,
```

Comment:
This renders as a single line despite the line breaks here (standard Markdown behavior). If that's unintended you need to add line breaks or define a list.

Reply:
Yes this is intended, I find it easier to edit.